test/test.go

```diff
@@ -524,9 +543,18 @@ func (h *Helper) Exist(path string) bool {
 
 // MustNotExist fails if path exists.
 func (h *Helper) MustNotExist(path string) {
+	if err := h.ShouldNotExist(path); err != nil {
+		h.t.Fatalf("%+v", err)
+	}
+}
+
+// ShouldNotExist returns an error if path exists.
+func (h *Helper) ShouldNotExist(path string) error {
```

Comment:
Hah, yes, totally. I'd forgotten about both that conversation, and consideration, despite being there for it 😄 Sorry, yes, let's carry on as-is.